scss/c2s/c2s.scss

```diff
@@ -1,6 +1,13 @@
 @import "betahealth/nhsuk";
 
 /*
+   Colours.
+*/
+$hiviz: #ffca55;
+$blue: #0072c6;
```

Comment:
If the variable is defined already within the `@import` does that make it available within this? It looks like it has been defined already in https://github.com/nhsuk/profiles/blob/db1e8a00ca69b5feb3b692a360d081670b2d9456/scss/betahealth/environment/settings/_colours.scss#L1